appium/api-mocks/apis/attester/attester-endpoints.ts

```diff
@@ -69,6 +74,11 @@ export const getMockAttesterEndpoints = (
     },
   };
 }
+const throwErrorIfConfigSaysSo = (config: AttesterConfig) => {
+    if (config.returnError) {
+        throw new FesHttpErr(config.returnError.message, config.returnError.code);
```

Comment:
Please declare `AttesterHttpError` in this file and use that, similar to how `FesHttpError` is defined.